### PR TITLE
Add readme section

### LINKED ISSUE #3: Add section on README
Add discussion of READMEs and other repository hygiene.

- LICENSES?
- Contribution guidelines
- As feels appropriate

Probably no exercise needed, but encourage users to go away and apply to their own repos?

### PR DESCRIPTION
**Pull request by MarionBWeinzierl**
Added section about ReadMes and other standard files in the repository, as well as licenses.

Closes #3 .

### CODE REVIEW

File: slides/_licenses.qmd [hunk 1]
```diff
@@ -0,0 +1,34 @@
+ 
+## How to choose a license
+ 
+
```
Comment (jatkinson1000) [inline slides/_licenses.qmd]: Do we want to briefly mention the copyleft - permissive spectrum, and the relative attributes of each?

File: slides/_licenses.qmd [hunk 2]
```diff
@@ -0,0 +1,34 @@
+ 
+## How to choose a license
+ 
+ 
+:::: {.columns}
+::: {.column width="70%"}
+ - [https://choosealicense.com/licenses/](https://choosealicense.com/licenses/)
```
Comment (jatkinson1000) [inline slides/_licenses.qmd:12]: Could also mention the ability to easily add licenses in GitHub/GitLab?

File: slides/_comments_docstrings.qmd [hunk 3]
```diff
@@ -272,3 +272,40 @@ Docstrings:
...
+ - Project/release status
+ - Requirements and environment
+ - Installation
+ - Building
+ - Usage/Quick start guide (incl. simple example with expected output)
+ - Documentation and Support: User Guide/Website/Help/Discussion forum
+ - Contributing guide
+ - Authors and Acknowledgement
+ - Licence
+ - Known Issues
```
Comment (jatkinson1000) [inline slides/_comments_docstrings.qmd:304]: Add links to [makeareadme.com](https://www.makeareadme.com/) and [readme.so](https://readme.so/)?

Mention markdown syntax? or is that information overload here perhaps?

Comment (jatkinson1000) [inline slides/_comments_docstrings.qmd:304]: We should probably ensure that our repo readme for this workshop meets most of these in #11 😅

**Review by jatkinson1000 — APPROVED** [review]
Thanks @MarionBWeinzierl 

This seems sensible to me.
A couple of additional comments for your consideration.

On images - If I think the url will be long-lived I tend to refer to it by link in the sides rather than uploading the image to the repo. The disadvantage here is that it may disappear at some point (has happened to me) or won't load without internet. So a matter of preference. I'm still not convinced that this is the 'correct' approach 🤷‍♀️

File: slides/_licenses.qmd (hunk 2 above)
Comment (MarionBWeinzierl) [reply slides/_licenses.qmd:12]: I added screenshots

File: slides/_licenses.qmd (hunk 1 above)
Comment (MarionBWeinzierl) [reply slides/_licenses.qmd]: I added a graphic -- could you check whether you think that's enough?

File: slides/_comments_docstrings.qmd (hunk 3 above)
Comment (MarionBWeinzierl) [reply slides/_comments_docstrings.qmd:304]: I think the most important parts are now there, but it could always be extended....

Comment (MarionBWeinzierl) [reply slides/_comments_docstrings.qmd:304]: Added it in

File: slides/_licenses.qmd (hunk 2 above)
Comment (jatkinson1000) [reply slides/_licenses.qmd:12]: Good idea!

**Review by jatkinson1000 — APPROVED** [review]
Yep, this looks great to me!
Happy for you to merge at will.

File: slides/_licenses.qmd (hunk 1 above)
Comment (jatkinson1000) [reply slides/_licenses.qmd]: I think that's good, choosealicense also shows this which we link to.
We can show the graphic and describe where key licenses (MIT, GPL) are.

This is much easier than my previous presentations where I tried to describe it. Why didn't I think of using a picture 😅 🤦

**Pull request merged**